packages/dashboard/src/components/FooterLine/About.tsx

```diff
@@ -0,0 +1,162 @@
+import { DiscordIcon, MaskBlueIcon, MaskGreyIcon, MaskTextIcon, MaskTextNightlyIcon } from '@masknet/icons'
+import { Avatar, IconButton, Link, makeStyles, Typography } from '@material-ui/core'
+import FacebookIcon from '@material-ui/icons/Facebook'
+import GitHubIcon from '@material-ui/icons/GitHub'
+import TelegramIcon from '@material-ui/icons/Telegram'
+import TwitterIcon from '@material-ui/icons/Twitter'
+import { useDashboardI18N } from '../../locales'
+import { experimentalStyled as styled } from '@material-ui/core/styles'
+import { Version } from './Version'
+import { getMaskColor } from '@masknet/theme'
+import links from './links.json'
+
+const useStyles = makeStyles((theme) => ({
+    wrapper: {
+        width: 580,
+        minHeight: 660,
+        lineHeight: 1.75,
+    },
+    header: {
+        height: 300,
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        background: `url(${new URL('./AboutDialogBackground.png', import.meta.url)}) no-repeat center / cover`,
```

Comment:
extract this `new URL` as constant.
```suggestion
        background: `url(${ABOUT_DIALOG_BACKGROUND}) no-repeat center / cover`,
```

Reply:
I created `packages/dashboard/src/assets/index.ts` to manage static resource.